pages/ksphere/konvoy/1.6/training/index.md

```diff
@@ -18,10 +18,3 @@ D2IQ provides training for all of the D2IQ products. Two types of training are a
   - Live instructor for Q&A
   - Content for intermediate and advanced learners
 
-- [Training using Conductor](https://d2iq.com/products/conductor)
```

Comment:
You need to also adjust line 13 in this page as it says "Two types of training" but now there will only be one section.   

This page is in the Konvoy repo; I don't understand how the Konvoy docs are merged into this repo so I am not sure you need to change this here.  If you do, note that there is already a Konvoy PR that touches this page:

https://github.com/mesosphere/konvoy/pull/2367

Reply:
@cneth @pierrebeitz - I'll push a commit for this, because we're already modifying these pages.